engine/include/cubos/engine/assets/bridge.hpp

```diff
@@ -46,5 +55,15 @@ namespace cubos::engine
         /// @param handle Handle of the asset being saved.
         /// @return Whether the asset was successfully saved.
         virtual bool save(const Assets& assets, const AnyAsset& handle);
+
+        /// @brief Gets the type of the assets the bridge loads.
+        /// @return The type_index of the asset.
```

Comment:
```suggestion
        /// @return Type of the asset.
```

Avoid unnecessary 'The's